### ISSUE TITLE
Exposure to aquatic animals

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10000]
**Preferred term label:**
Exposure to aquatic animals

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mycobacterium marinum infection

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000055
- label: Exposure to aquatic animals
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: PMID:29493404
- def: History of exposure to aquatic animals.
- comment: Mycobacterium marinum is an opportunistic pathogen inducing infection in fresh and marine water fish. Human infections with Mycobacterium marinum have been associated with the exposure of damaged skin to polluted water from fish pools or contacting objects contaminated with infected fish. Fish mycobacteriosis is clinically manifested and characterized in man by purple painless nodules, liable to develop into superficial crusting ulceration with scar formation.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10000
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
thanks for pointing this out. I am using a new program to accelerate the creation of HPO terms. Something seems to have gone wrong.
- [ ] Check: HP:6000055 through HP:6000069 seem to have been used twice.

**Comment by pnrobinson** [comment]
@MickeySegal 
Fixed this:
Exposure to aquatic animals HP:6000055
Nipple localization HP:0040402

It will probably be prudent to check all IDs when we are finished, but I think errors like this have been rare.
